setup.cfg

```
@@ -1,41 +1,85 @@
 [metadata]
 name = pythonrepo
-version = 1.1.4
+version = 1.1.5
 author = Mr. Walls
-author-email = reactive-firewall@users.noreply.github.com
-summary = python-repo template
-description = file: README.md
-home-page = https://github.com/reactive-firewall/python-repo
-download-url = https://github.com/reactive-firewall/python-repo.git
-license = file: LICENSE.md
+author_email = reactive-firewall@users.noreply.github.com
+description = python-repo template
+long_description = file:README.md
+long_description_content_type = text/markdown
+url = https://github.com/reactive-firewall/python-repo
+download_url = https://github.com/reactive-firewall/python-repo.git
 classifiers =
-	Development Status :: 4 - Beta
-	Operating System :: POSIX :: Linux
-	Programming Language :: Python
-	Programming Language :: Python :: 3
-	Programming Language :: Python :: 3.12
-	Programming Language :: Python :: 3.11
-	Programming Language :: Python :: 3.10
-	Programming Language :: Python :: 3.9
-	Programming Language :: Python :: 3.8
-	Programming Language :: Python :: 3.7
-	Programming Language :: Python :: 3.6
-	Programming Language :: Python :: 3.5
-	Programming Language :: Python :: 3.4
-	Programming Language :: Python :: 3.3
-	Programming Language :: Python :: 2.7
-	Topic :: Security
+  Development Status :: 4 - Beta
+  Operating System :: MacOS :: MacOS X
+  Operating System :: POSIX :: Linux
+  Programming Language :: Python
+  Programming Language :: Python :: 3
+  Programming Language :: Python :: 3.14
+  Programming Language :: Python :: 3.13
+  Programming Language :: Python :: 3.12
+  Programming Language :: Python :: 3.11
+  Programming Language :: Python :: 3.10
+  Programming Language :: Python :: 3.9
+  Programming Language :: Python :: 3.8
+  Programming Language :: Python :: 3.7
+  Programming Language :: Python :: 3.6
+  Programming Language :: Python :: 3.5
+  Programming Language :: Python :: 3.4
+  Programming Language :: Python :: 3.3
+  Programming Language :: Python :: 2.7
+  Topic :: Software Development :: Libraries :: Python Modules
+  Topic :: Security
+license = MIT
+license_files =
+  LICENSE[.md]*
+
+platform = any
+project_urls =
+  Bug Tracker = https://github.com/reactive-firewall/python-repo/issues
+  License = https://github.com/reactive-firewall/python-repo/LICENSE.md
+
+[bdist_rpm]
+url = https://github.com/reactive-firewall/multicast.git
 
 [bdist_wheel]
 universal=1
 
-[files]
-packages = pythonrepo
-
```

🤔 on line-by-line review this does not seem nessacery to remove?

```suggestion
[files]
packages = pythonrepo

```

@coderabbitai, am I mistaken or should it be safe to re add this? can you check the official documentation of setup.cfg configuration files. Please consider cross-python compatibility for versions 2.7-3.13.